### ISSUE TITLE
Total Install Size not calculating for not installed games in "Statistics" Tab

### ISSUE THREAD

**Issue by FlyMandi** [JosefNemec/Playnite#3664]
### Bug Description

Statistics not calculating Install Size for uninstalled games.
If I filter by "Not Installed" it shows the total install size to be 0B (which is false, I have lots and lots of uninstalled games with install size > 0B in their metadata).
If I filter by only "installed" it shows this exact value, so my guess is that this statistics field only sums the install size of currently installed games. Just to check, I set a big game like Forza 5 to "uninstalled" and low and behold, the total install size for all games drops down from 3.58 TB to 3.414 TB. This is all _without_ editing the metadata of any game.

### Diagnostics ID

b93080f2-307a-4701-bb4c-0d0def755c47

### Screenshots

![image](https://github.com/JosefNemec/Playnite/assets/115730623/5b4d5c6c-c20f-4ead-a6df-f0ae7f22229a)

**Comment by JosefNemec** [comment]
This is intentional because install size value for games that are not installed can't be reliable or accurate.

**Comment by FlyMandi** [comment]
Can we have a setting to calculate not installed games? even if that estimate is bad, I'd want to know (roughly) how much it takes to install all of my games.

Also, if the game ever was installed, the information is pretty reliable. I'd still want that as a feature, if possible.

**Comment by FinMckinnon** [comment]
I agree here, I'd like to see something like Game Size Calculator implemented to calculate my entire library size so I know how much storage I'd need for all my games etc.
